scripts/patch-realsense-ubuntu-lts-hwe.sh

```diff
@@ -71,7 +71,7 @@ source ./scripts/patch-utils-hwe.sh
 LINUX_BRANCH=${LINUX_BRANCH:-$(uname -r)}
 
 # Get the required tools and headers to build the kernel
-sudo apt-get install linux-headers-generic linux-headers-$LINUX_BRANCH build-essential git bc -y
+sudo apt-get install linux-headers-$LINUX_BRANCH build-essential git bc -y
```

Comment:
Removed installing `linux-headers-generic`. Since, it is in development, new versions are released regularly and it leads to installing a kernel version greater than the local kernel version. For example, default kernel for U24 is 6.8.0-**31** and this command installs 6.8.0-**40**.